api/src/main/resources/emailTemplate.properties

```diff
@@ -0,0 +1,2 @@
+mail.subject=password reset request
+mail.content=Dear :name \n You recently requested a password reset for your OpenMRS account. To complete the process, click the link below.\n  Please disregard this email if you didn't innitiate the password reset process.  \n 
```

Comment:
Change **Dear** to **Hi**

Is name a place holder? I would rather have it as **{name}** rather than **:name**

Reply:
@fanyui  The `\n` character is for the terminal outputs, will it work for the HTML based email content? Can you check and confirm with `<br/>`?

This link will active for some amount of time (24hrs). Right? So better to mention that at the end of this content.

Reply:
That's what I think too, we should use html.